portal-web/test/functional/com/liferay/portalweb/macros/WorkflowMetrics.macro

```diff
@@ -759,4 +782,22 @@ definition {
 		}
 	}
 
+	macro waitForSLA {
+		var count = "0";
+
+		while (IsTextEqual(locator1 = "WorkflowMetrics#DASHBOARD_TOTAL_${key_status}_ITEMS_VALUE", value1 = "${key_expectedValue}") && ("${count}" != "${timesToRun}")) {
+
+			echo("Waiting 5 seconds to recheck");
+
+			Pause(locator1 = "5000");
+
+			WorkflowMetrics.goToWorkflowProcessMetrics(workflowProcessName = "Single Approver");
+
+		}
+
+		if ("${count}" == "${timesToRun}") {
+			echo("timeout exceeded");
+		}
+	}
```

Comment:
Hi @Adriverforlife 
Does this macro replaces waitForCompletedSLAtoBecomeOverdue and waitForSLAtoBecomeOverdue? If yes, would you please delete the deprecated macros and replace the test cases that should use this new macro?

Reply:
It would be able to replace waitForSLAtoBecomeOverdue. I could write a new version of waitForCompletedSLAtoBecomeOverdue based on this one,

Reply:
If you think that it is going to take a lot of time for you to rewrite waitForCompletedSLAtoBecomeOverdue, you can edit only waitForSLAtoBecomeOverdue and edit waitForCompletedSLAtoBecomeOverdue later.

Reply:
If I remember correctly only big difference is the navigation steps in them. If I am already there it should be an easy enough update and be much more useful i the future.

Reply:
Thank you!